tests/kafkatest/tests/core/transactions_test.py

```diff
@@ -137,16 +119,15 @@ def bounce_copiers(self, copiers, clean_shutdown):
                                                         str(copier.progress_percent())))
                 copier.restart(clean_shutdown)
 
-    def create_and_start_copiers(self):
+    def create_and_start_copiers(self, input_topic, output_topic, num_copiers):
         copiers = []
-        copiers.append(self.create_and_start_message_copier(
-            input_partition=0,
-            transactional_id=self.first_transactional_id
-        ))
-        copiers.append(self.create_and_start_message_copier(
-            input_partition=1,
-            transactional_id=self.second_transactional_id
-        ))
+        for i in range(0, num_copiers):
+            copiers.append(self.create_and_start_message_copier(
+                input_topic=input_topic,
+                output_topic=output_topic,
+                input_partition=i,
+                transactional_id="transactional-copier-" + str(i)
```

Comment:
nit: wonder if we could make this more concise since it shows up in all of the log mesages. Maybe drop the "transactional-" prefix?